docs/ref/forms/validation.txt

```diff
@@ -366,6 +366,9 @@ example::
 
 In this code, if the validation error is raised, the form will display an
 error message at the top of the form (normally) describing the problem.
+Note that errors generated here will be form errors and not field errors,
+to display these errors you'll have to define ``{{ form.non_field_errors }}``
+in your template.
```

Comment:
I edited down but don't think I lost any information:

```suggestion
Such errors are non-field errors, which are displayed in the template with ``{{ form.non_field_errors }}``.
```

Reply:
Thanks @adamchainz, this is much better. :)